frontend/src/views/Pages/AccountOverview/GddSend.vue

```diff
@@ -254,6 +254,7 @@ export default {
       const result = await communityAPI.send(
         this.$store.state.session_id,
         this.ajaxCreateData.email,
+        // better to send the user's input here?
         this.ajaxCreateData.amount * 10000,
```

Comment:
Will you adjust the send call in this PR aswell? @einhornimmond

Reply:
yes